ansible/requirements.yml

```diff
@@ -0,0 +1,133 @@
+# External Ansible roles required by this repository
+
+- src: openmicroscopy.omero-common
+  version: 0.1.0
+
+- src: openmicroscopy.analysis-tools
+  version: 1.0.0
+
+- src: openmicroscopy.basedeps
+  version: 1.0.0
+
+- src: openmicroscopy.cli-utils
+  version: 1.0.0
+
+- src: openmicroscopy.docker
+  version: 1.0.0
+
+- src: openmicroscopy.docker-tools
+  version: 1.0.0
+
+- src: openmicroscopy.haproxy
+  version: 1.0.0
+
+- src: openmicroscopy.ice
+  version: 1.0.0
+
+- src: openmicroscopy.java
+  version: 2.0.0
+
+- src: openmicroscopy.jekyll-build
+  version: 1.2.1
+
+- src: openmicroscopy.local-accounts
+  version: 1.0.0
+
+- src: openmicroscopy.logrotate
+  version: 1.0.0
+
+- src: openmicroscopy.lvm-partition
+  version: 1.1.0
+
+- src: openmicroscopy.munin
+  version: 1.1.3-openmicroscopy1
+
+- src: openmicroscopy.munin-node
+  version: 1.2.1-openmicroscopy1
+
+- src: openmicroscopy.network-cloud-interfaces
+  version: 1.1.0
+
+- src: openmicroscopy.nfs-mount
+  version: 1.0.0
+
+- src: openmicroscopy.nfs-share
+  version: 1.0.0
+
+- src: openmicroscopy.nginx
+  version: 1.0.0
+
+- src: openmicroscopy.nginx-proxy
+  version: 1.1.0
+
+- src: openmicroscopy.nginx-ssl-selfsigned
+  version: 1.0.0
+
+- src: openmicroscopy.omero-logmonitor
+  version: 1.0.0
+
+- src: openmicroscopy.omero-python-deps
+  version: 1.0.0
+
+- src: openmicroscopy.omero-server
+  version: 1.2.0
+
+- src: openmicroscopy.omero-web-runtime
+  version: 1.1.0
+
+- src: openmicroscopy.openstack-volume-storage
+  version: 1.0.0
+
+- src: openmicroscopy.postgresql
+  version: 2.0.0
+
+- src: openmicroscopy.python-pydata
+  version: 1.0.0
+
+- src: openmicroscopy.redis
+  version: 1.0.0
+
+- src: openmicroscopy.rsync-server
+  version: 1.0.0
+
+- src: openmicroscopy.samba-client
+  version: 1.0.0
+
+- src: openmicroscopy.selinux-utils
+  version: 1.0.1
+
+- src: openmicroscopy.storage-volume-initialise
+  version: 1.0.0
+
+- src: openmicroscopy.sudoers
+  version: 1.0.0
+
+- src: openmicroscopy.upgrade-distpackages
+  version: 1.0.0
+
+- src: openmicroscopy.versioncontrol-utils
+  version: 1.0.0
+
+# External development roles
+
+- src: openmicroscopy.omero-user
+  version: 0.1.0
+
+- src: openmicroscopy.hosts-populate
+  version: 0.1.0
+
+- src: openmicroscopy.mysql-backup
+  version: 0.1.0
+
+- src: openmicroscopy.omero-web-apps
+  version: 0.1.0
+
+- src: openmicroscopy.reboot-server
+  version: 0.1.0
+
+- src: git+https://github.com/jburel/ansible-role-devspace.git
```

Comment:
@jburel now everything has been tested by @pwalczysko  and https://github.com/openmicroscopy/ansible-role-devspace/pull/5 has been integrated, can you push another commit to point at the master branch of the upstream repo for now